### DISCUSSION TITLE
Execute-command-if-idle

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#737] [__Dismissed__] [answered]
make it so if the player is idle he automatically executes a command. this would be usefull so players can get moved to a limbo server and if they move they automatically get back to the server they were on. i need this feature cause i am using tcpshield and with this option i would save a lot of bytes on tcpshield

**Comment by Tizooo** [comment]
or a option to make it so they only send data/packets they need to stay connected till they move again

**Comment by BillyGalbreath** [comment, marked as answer]
We have an event so you can create a plugin to handle stuff like that. [PlayerAFKEvent](https://purpurmc.org/javadoc/org/purpurmc/purpur/event/PlayerAFKEvent.html)
